README.md

```diff
@@ -157,6 +158,22 @@ yarn start
 
 The app and API are both served on  [http://localhost:3345/](http://localhost:3345/)
 
+## Deployment to Vonage Cloud Runtime
+
+You can easily deploy your local branch to Vonage Cloud Runtime (VCR) using the tools in this repository. See https://developer.vonage.com/en/vonage-cloud-runtime/overview for an overview of Vonage Cloud Runtime.
+
+Firstly, install the VCR cli: https://developer.vonage.com/en/vonage-cloud-runtime/getting-started/working-locally#cli-installation and run `vcr configure`.
+
```

Comment:
I think we're missing one step which is running vcr init to create an instance and an application for VCR to use for deployment. Let's huddle later